web/source/keymanweb.js

```diff
@@ -3183,14 +3193,27 @@ if(!window['keyman']['initialized']) {
           return false;
         }
       }
+
+      if(typeof(Levent.Ltarg.base) != 'undefined') {
+        // Simulated touch elements have no default text-processing - we need to rely on a strategy similar to
+        // that of the OSK here.
+        var ch = osk.defaultKeyOutput('',Levent.Lcode,Levent.Lmodifiers,false,Levent.Ltarg);
+        if(ch) {
+          kbdInterface.output(0, Levent.Ltarg, ch);
+          return false;
+        }
+      }
       return true;
     }                
 
     keymanweb.callKeyboardStartGroup = function(Ltarg, Levent) {
```

Comment:
Can we document who calls this? Is there any reason why this is only called for hardware keyboard input processing? The name doesn't signify that...

Reply:
Ran a check with it; I think `callKeyboardStartGroup` was the original `processKeystroke` but was never utilized for OSK input for some reason.  Things seem pretty stable when merging the two methods.

I may run with `processKeystroke` as the name since it's less wordy and leave an internal comment about calling the start group.